### DISCUSSION TITLE
Will the outgoing and back link panes support canvas?

### DISCUSSION THREAD

**Discussion by PennHui666** [RyotaUshio/obsidian-pdf-plus#90] [Q&A]
I like to extract information from PDF files and organize it into canvas, so I think it would be great if the outgoing and back link panes could support canvas files.

**Comment by RyotaUshio** [comment]
I absolutely agree with you, but unfortunately, it's not something a plugin can address.
Since it's related to Obsidian's core linking system, it can only be addressed by Obsidian itself.

The good new is there is a feature request thread on the forum, which is created by a member of the Obsidian team:

https://forum.obsidian.md/t/canvas-cards-and-notes-in-links-backlinks-and-outgoing-arrows-in-graph-view/49697
